packages/std/examples/schema.rs

```diff
@@ -13,20 +13,37 @@ fn main() {
 
     export_schema(&schema_for!(Env), &out_dir);
     export_schema(&schema_for!(CosmosMsg), &out_dir);
-    export_schema(&schema_for!(InitResult), &out_dir);
-    export_schema(&schema_for!(HandleResult), &out_dir);
-    export_schema(&schema_for!(QueryResult), &out_dir);
+    export_schema_with_title(&mut schema_for!(InitResult), &out_dir, "InitResult");
+    export_schema_with_title(&mut schema_for!(HandleResult), &out_dir, "HandleResult");
+    export_schema_with_title(&mut schema_for!(QueryResult), &out_dir, "QueryResult");
```

Comment:
Hmm, it is getting more complicated again. But I can see why this is needed. I guess it is time to create a dev dependency cosmwasm-schema for those implementations.

Reply:
It's just a few lines and it overrides the json title as well as filename